scripts/bundleSize.js

```diff
@@ -13,117 +15,159 @@ const { MIN_SIZE, MAX_SIZE } = require('./bundleSizeConfig');
+const averageBundleSize = serviceBundlesTotals.reduce(
+  (acc, currentValue, currentIndex, array) => {
+    const isLastItem = currentIndex === array.length - 1;
+    const total = acc + currentValue;
+    if (isLastItem) {
+      return Math.round(total / array.length);
+    }
+    return total;
+  },
+  0,
+);
+const smallestBundleSize = Math.min(...serviceBundlesTotals);
+const largestBundleSize = Math.max(...serviceBundlesTotals);
 
-console.log(''); // eslint-disable-line no-console
+console.log('');
 const spinner = ora({
   text: 'Analysing bundles...',
   color: 'magenta',
 });
 spinner.start();
 
-let totalSize = 0;
-let smallestBundle;
-let largestBundle;
+const serviceBundlesTable = new Table({
+  head: [
+    'Service name',
+    'Service bundle sizes (kB)',
+    'Total service bundle sizes (kB)',
+    'Main bundle size (kB)',
+    'Vendor bundle size (kB)',
+    'Total bundles size for service (kB)',
+  ],
+});
 
-const errors = services
-  .map((service) => {
-    const size = getServiceBundleSize(service);
+const tableHead = [
+  'Bundle name',
+  'Bundle sizes (kB)',
+  'Total bundles size (kB)',
+];
 
-    totalSize += size;
+const mainBundlesTable = new Table({
+  head: tableHead,
+});
 
-    if (!smallestBundle || size < smallestBundle.size) {
-      smallestBundle = { service, size };
-    }
+const vendorBundlesTable = new Table({
+  head: tableHead,
+});
 
-    if (!largestBundle || size > largestBundle.size) {
-      largestBundle = { service, size };
+const pageBundlesTable = new Table({
+  head: tableHead,
+});
+
+const summaryTable = new Table({
+  head: [
+    'Average total bundle size (kB)',
+    'Smallest total bundle size (kB)',
+    'Largest total bundle size (kB)',
+  ],
+});
+
+const sortByTotalAscending = bundlesData =>
+  bundlesData.sort((a, b) => {
+    const total1 = a[a.length - 1];
+    const total2 = b[b.length - 1];
+
+    return total1 - total2;
+  });
+
+sortByTotalAscending(serviceBundleData).forEach(bundle =>
+  serviceBundlesTable.push(bundle),
+);
+mainBundleData.forEach(bundle => mainBundlesTable.push(bundle));
+vendorBundleData.forEach(bundle => vendorBundlesTable.push(bundle));
+sortByTotalAscending(pageBundleData).forEach(bundle =>
+  pageBundlesTable.push(bundle),
+);
+summaryTable.push([averageBundleSize, smallestBundleSize, largestBundleSize]);
+
+console.log('\n\nResults');
+console.log(serviceBundlesTable.toString());
+console.log(mainBundlesTable.toString());
+console.log(vendorBundlesTable.toString());
+console.log(pageBundlesTable.toString());
+console.log(summaryTable.toString());
+
+const errors = serviceBundlesTotals
+  .map((size, index) => {
+    const serviceName = serviceBundleData[index][0];
+    if (size < MIN_SIZE) {
+      return createConsoleError(serviceName, size, 'small');
     }
 
-    return mapSizeToError(service, size);
+    if (size > MAX_SIZE) {
+      return createConsoleError(serviceName, size, 'large');
+    }
```

Comment:
Should we consider tracking the bundle sizes for each page type and failing CI if they get too big? Taking the largest total bundle size from the PR description and adding it to the Story page as an example gives 
`665 + 230 = 895`; this is actually larger than our max bundle size now, have I misunderstood something?